modules/DesktopNavigation.js

```diff
@@ -0,0 +1,25 @@
+import {
+  hamburger, dropDownNav, listNav, addNewNav, contactNav,
+} from './Initialisations.js';
+
+export default () => {
+  // Query selector for mobile navigation
+  if (window.matchMedia('(min-width: 768px)').matches) {
+    hamburger.classList.add('hide');
+    dropDownNav.classList.remove('hide');
+
+    // List event listener
+    listNav.addEventListener('click', () => {
+      dropDownNav.classList.remove('hide');
+    });
+
+    // New book event listener
+    addNewNav.addEventListener('click', () => {
+      dropDownNav.classList.remove('hide');
+    });
+    // Contact section event listener
+    contactNav.addEventListener('click', () => {
+      dropDownNav.classList.remove('hide');
+    });
```

Comment:
- [ ] Nice work. But based on project requirements, you're meant to display the date which you get using the `luxon` module below the navigation bar of your UI. Kindly follow this instruction as specified 👍 

## view
![image](https://user-images.githubusercontent.com/50990925/235442430-f6ed50df-3bca-454a-8d8a-7b2602d5e3f6.png)

## Project requirements:
![image](https://user-images.githubusercontent.com/50990925/235442516-75e0aa23-cfef-48b8-a50b-1214da1e5747.png)